staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/validating/dispatcher.go

```diff
@@ -126,6 +133,11 @@ func (d *validatingDispatcher) Dispatch(ctx context.Context, attr admission.Attr
 			if callErr, ok := err.(*webhookutil.ErrCallingWebhook); ok {
 				if ignoreClientCallFailures {
 					klog.Warningf("Failed calling webhook, failing open %v: %v", hook.Name, callErr)
+
+					key := hook.Name + "/" + ValidatingWebhookFailedOpenKeySuffix
```

Comment:
this name would collide with a `failed-open` audit annotation returned by the webhook... that does not seem desireable

Reply:
Are you referring to the domain name or the `failed-open` suffix?

For the domain name, the webhook name was put there, instead of a k8s domain name, to distinguish on which webhook we failed open on, as multiple webhooks can be configured.

For the suffix, this is mainly to be backwards compatible with the ImagePolicyWebhook annotation key we've been using (https://github.com/kubernetes/kubernetes/blob/323f34858de18b862d43c40b2cced65ad8e24052/plugin/pkg/admission/imagepolicy/admission.go#L58). And IMO, fail-open is something that _should_ be handled by the client but not the webhook, so I would imagine it's fairly unlikely the webhook to write a failed-open audit annotation?

Reply:
> Are you referring to the domain name or the `failed-open` suffix?

The combination of the two... using the webhook name as the domain name means our `failed-open` annotation suffix is in a domain I would expect to be under the control of the webhook, not used by the kube-apiserver machinery.

Reply:
I changed to use "validation.webhook.admission.k8s.io/failed-open" and "mutation.webhook.admission.k8s.io/failed-open"